CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every little bit
+helps, and credit will always be given.
+
+## Get Started!
+Ready to contribute? Here's how to set up `pylibczi` for local development.
+
+1. Fork the `pylibczi` repo on GitHub.
+2. Clone your fork locally:
+
+    $ git clone git@github.com:{your_name_here}/pylibczi.git
+
+3. Install the project in editable mode. (It is also recommended to work in a virtualenv or anaconda environment):
+
+    $ cd pylibczi/
+    $ pip install -e .[dev]
+
+4. Create a branch for local development:
+
+    $ git checkout -b {your_development_type}/short-description
+
+    Ex: feature/read-tiff-files or bugfix/handle-file-not-found
+
+    Now you can make your changes locally.
+
+5. When you're done making changes, check that your changes pass linting and
+   tests, including testing other Python versions with tox:
+
+    $ tox
```

Comment:
I have not used this before, so this might be a stupid question. Do we have a config file that is checked into the repository, or the user supplies a config file. Just running as it, for me it fails with this error:
```
ERROR: tox config file (either pyproject.toml, tox.ini, setup.cfg) not found
```

Reply:
I think @heeler just copied the standard contributing doc from our cookiecutter python package. Should change it to be specific to this repo or add a `tox.ini` file.

`tox` just lets us run tests always in a clean environment for both 3.6 and 3.7 (or w.e. python version you want)

Reply:
Jackson is right I just copied a default. I'm adding a tox.ini file for the project now and you can just that to run the local tests.